tasks/compile.py

```diff
@@ -15,8 +15,7 @@
 def _copy_built_function(user, func):
     dest_folder = join(WASM_DIR, user, func)
 
-    if not exists(dest_folder):
-        mkdir(dest_folder)
+    makedirs(dest_folder, exist_ok=True)
```

Comment:
Nice, didn't know that existed. There a few places I've done the `if not exists...` thing so I'll keep an eye out :eyes: